programs/compressed-pda/src/sdk.rs

```diff
@@ -24,9 +27,7 @@ pub fn create_execute_compressed_instruction(
     payer: &Pubkey,
     input_compressed_accounts: &[CompressedAccount],
     output_compressed_accounts: &[CompressedAccount],
-    input_compressed_account_merkle_tree_pubkeys: &[Pubkey],
-    input_compressed_accounts_leaf_indices: &[u32],
-    nullifier_queue_pubkeys: &[Pubkey],
+    merkle_context: &[MerkleContext],
```

Comment:
Would it make sense to include the output MT in this packed struct too?

Reply:
Imo not in this one but we can consider to add a context object for the output compressen accounts as well.
So far I haven’t because the current structure is easy to use internally onchain.